src/db/migration/migrate20220729-FilesShareWithMeFolder.pl

```diff
@@ -81,14 +86,14 @@ ($)
     mod_metadata, change_date, mod_content
 )
 SELECT
-    id, $ID, 1, 1, 1, null, null,
-    $NOW, 0, null,
+    id, $folderid, 1, 1, 1, null, null,
+    $now, 0, null,
     0, 0, 0, null,
-    '$FOLDERNAME', '$FOLDERNAME', '$METADATA',
-    change_checkpoint, $NOW, change_checkpoint
+    '$foldername', '$foldername', '$metadata',
+    change_checkpoint, $now, change_checkpoint
 FROM mailbox
 WHERE group_id = $gid
-ON DUPLICATE KEY UPDATE name = '$FOLDERNAME';
+ON DUPLICATE KEY UPDATE name = '$foldername';
```

Comment:
Unfortunate we're interpolating here instead of using bind variables, but fixing that would require a new `runSqlParallel`, and since they're all constants here it isn't (right now) a problem.

Reply:
this is first pull request from me this repo, so have zero idea about this :)